frontend/src/global_styles/content/user-content/_table.sass

```diff
@@ -18,17 +18,16 @@
     padding: 0.75rem
 
     &:not(:last-child):not([colspan])
-      border-right: 1px solid var(--table-border-color)
+      border-right: 1px solid var(--borderColor-default)
 
     @include user-content-children
 
     @at-root .op-uc-container_editing &:hover
       background: rgba(26, 103, 163, 0.05)
 
     &_head
-      background: rgb(242, 242, 242)
-      background-clip: padding-box
-      border-bottom: 1px solid rgba(0, 0, 0, 0.1)
+      background: var(--display-gray-bgColor-muted)
+      border-bottom: 1px solid var(--borderColor-default)
```

Comment:
Both of these styles are overriden and not applied at all. Because of that, the headers are not shown differently at all in the dark mode (when editing e.g. the description)
<img width="1204" alt="Bildschirmfoto 2024-07-17 um 12 12 57" src="https://github.com/user-attachments/assets/6a55ed10-8126-4762-9ad3-4592143122a0">